src/lib/compressImage.ts

```diff
@@ -0,0 +1,33 @@
+// Compress image to 248x248 using canvas
+export default async function compressImage(file: Blob): Promise<Blob> {
```

Comment:
tip: Personally I avoid default imports for a few reasons:

- Devs can import it with any name which might make it difficult to find or make future migrations.
- It enforces consistent naming everywhere, improving readability/cognitive effort.